### DISCUSSION TITLE
How to remove border and background from Expander

### DISCUSSION THREAD

**Discussion by CollinAlpert** [AvaloniaUI/Avalonia#7733] [Q&A] [answered]
I am trying to style an `Expander` but am having trouble setting styles for the expander's content. Specifically I am trying to remove the border and set a transparent background. Any help on this would be greatly appreciated. I couldn't find anything about this in the docs.

**Comment by maxkatz6** [comment, marked as answer]
You can define your styles.
You have a choice either to change properties with specific selectors or set new template for the whole control.
For existing styles see https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/Expander.xaml

**Comment by ajmcateer** [comment]
Sorry to revive an old thread but I am trying to do the exact same thing and unable to get this selector right. How can I get the whole dropdown part to be the same color

```
<Expander Header="Utilities"
	Background="transparent"
                            BorderThickness="0"
	BorderBrush="transparent"
                            IsExpanded="True">
      <Expander.Styles>
	      <Style Selector="Expander:down /template/ Border#ExpanderContent">
		      <Setter Property="BorderThickness" Value="0"/>
	      </Style>
	      <Style Selector="Expander /template/ ContentPresenter">
		      <Setter Property="Background" Value="Red"/>
	      </Style>
      </Expander.Styles>
      <StackPanel Margin="0,-25,0,0">
	      <MenuItem Header="Bearer Token"
                                            Command="{Binding BearerToken}"
                                            Margin="1"/>
	      <MenuItem Header="Manager"
                                            Command="{Binding AwsToken}"
                                            Margin="1"/>
      </StackPanel>
      </Expander>
```

![image](https://user-images.githubusercontent.com/7401629/193885700-6b6d4657-2bf8-48e5-aead-5b1d24cc182f.png)

**Reply by rabbitism** [reply]
Expander fluent theme template has a default padding (16) for Border#ExpanderContent, you need to overwrite this. 

Or in your scenario, you should set Border#ExpanderContent background to red directly.

**Reply by ajmcateer** [reply]
@rabbitism I tried using these styles but it still shows as above

```
<Style Selector="Expander:expanded /template/ Border#ExpanderContent">
    <Setter Property="BorderThickness" Value="0"/>
</Style>
```
```
<Style Selector="Expander:expanded /template/ Border#ExpanderContent">
    <Setter Property="Background" Value="#202020"/>
</Style>
```

**Reply by rabbitism** [reply]
It works for me. 
![image](https://user-images.githubusercontent.com/14807942/195128574-2961118b-1e5c-42aa-a589-4b270d4f4e8a.png)

**Reply by ajmcateer** [reply]
I have it in the <Expander.Styles> block. It itsn't working is there something I have to adjust when in there. I added it to the windows.style area as well and still nothing.

```
<Expander.Styles>
	<Style Selector="Expander:expanded /template/ Border#ExpanderContent">
		<Setter Property="BorderThickness" Value="0"/>
	</Style>
	<Style Selector="Expander:expanded /template/ Border#ExpanderContent">
		<Setter Property="Background" Value="#202020"/>
	</Style>
	<Style Selector="Expander:expanded /template/ ContentPresenter">
		<Setter Property="Background" Value="#202020"/>
	</Style>
</Expander.Styles>
```

**Reply by MrOldOwl** [reply]
@ajmcateer I have such a problem too. At the same time, if you change "BorderThickness", it will display correctly, but it does not work with "Background" and "BorderBrush".
